### PR TITLE
[unity-js-bridge] add allChains to unity bridge wallet instantiation

### PR DESCRIPTION
**Pull request by 0xFirekeeper**

### CODE REVIEW

**Review by joaquim-verges — APPROVED** [review]
weird things happening with the pnpm lock - cc @jnsdls

File: packages/unity-js-bridge/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@thirdweb-dev/sdk": "workspace:*",
     "@thirdweb-dev/storage": "workspace:*",
     "@thirdweb-dev/wallets": "workspace:*",
+    "@thirdweb-dev/chains": "workspace:*",
```
Comment (jnsdls) [inline packages/unity-js-bridge/package.json:20]: why remove this here? it's still being used no?

**Review by jnsdls — COMMENTED** [review]
1 question rest makes sense to me

File: packages/unity-js-bridge/package.json (hunk 1 above)
Comment (jnsdls) [reply packages/unity-js-bridge/package.json:20]: wait I'm stupid it's being added ignore me

**Comment by jnsdls** [comment]
> weird things happening with the pnpm lock - cc @jnsdls

yeah not sure what's going on but let me pull it and resolve issues anyways and see

**Comment by jnsdls** [comment]
can't find issues with it no idea why the lock file changed so much - ship

**Pull request merged**